integrations/wagmi/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import { wrapEIP1193Provider, wrap } from '@oasisprotocol/sapphire-paratime';
+import type { Transport } from '@wagmi/core';
+import { injected, custom } from '@wagmi/core';
+
+type Window = {
+  ethereum?: any;
```

Comment:
Avoid `any`

Reply:
For that matter, avoid any everywhere in this file.

Reply:
This is unavoidable in only one instance, which remains.

Because the `viem` EIP1193Provider type is so specific it's incompatible with any other type definition of an EIP1193 provider.

Reply:
Looks good now 👍